virtcontainers/kata_agent.go

```diff
@@ -54,6 +54,8 @@ const (
 
 	// path to vfio devices
 	vfioPath = "/dev/vfio/"
+
+	agentPidEnv = "AGENT_PIDNS"
```

Comment:
`KATA_AGENT_PIDNS` (here and in the comments in the config files)?

Reply:
done @jodh-intel